src/assets/templates/widgets/finances_links.html

```diff
@@ -90,8 +90,16 @@ <h3 data-ng-bind="subcategory"></h3>
         </li>
         <li data-ng-if="subcategory === 'Financial Assistance' && financialAidSummaryLink.url && canViewFinancialAidSummaryLink">
           <a data-cc-campus-solutions-link-directive="financialAidSummaryLink"
+            data-cc-campus-solutions-link-directive-name="financialAidSummaryLink.name"
+            data-cc-campus-solutions-link-directive-title="financialAidSummaryLink.title"
+            data-cc-campus-solutions-link-directive-url="financialAidSummaryLink.url"
+            data-cc-campus-solutions-link-directive-cc-cache="finaid"
             data-cc-campus-solutions-link-directive-cc-page-name="currentPage.name"
             data-cc-campus-solutions-link-directive-cc-page-url="currentPage.url"
+            data-cc-campus-solutions-link-directive-uc-from="financialAidSummaryLink.isCsLink"
+            data-cc-campus-solutions-link-directive-uc-from-link="financialAidSummaryLink.isCsLink"
+            data-cc-campus-solutions-link-directive-uc-from-text="financialAidSummaryLink.isCsLink"
```

Comment:
@marcpechaitis - Does the `financialAidSummaryLink` object itself not include the name, title, url, etc?

We only need to specify those on the directive if we're trying to over-ride a specific value that's in the object, or if such values are not in the object for whatever reason.

Reply:
Thanks for the call out @redconfetti ... I am able to handle a lot of these via the Link API config data.